src/app/backend/search/search.go

```diff
@@ -70,6 +62,18 @@ type SearchResult struct {
 	PodList                   pod.PodList                  `json:"podList"`
 	DaemonSetList             daemonset.DaemonSetList      `json:"daemonSetList"`
 	StatefulSetList           statefulset.StatefulSetList  `json:"statefulSetList"`
+}
+
+// SearchResult is a list of resources matching search criteria found in whole cluster.
+type SearchResult struct {
+	ResourcesInNamespace
```

Comment:
I think we should keep `json:...` annotation as it's most common pattern for us.

@floreks Opinion?